octopoes/nibbles/https_availability/nibble.py

```diff
@@ -0,0 +1,66 @@
+from nibbles.definitions import NibbleDefinition, NibbleParameter
+from octopoes.models import Reference
+from octopoes.models.ooi.network import IPAddress, IPPort
+from octopoes.models.ooi.web import Website
+
+
+def query(targets: list[Reference | None]) -> str:
+    def pull(statements: list[str]) -> str:
+        return f"""
+                {{
+                    :query {{
+                        :find [(pull ?ipaddress [*]) (pull ?ipport80 [*]) (pull ?website [*]) (count ?ipport443)]
+                        :where [
+                            {" ".join(statements)}
+                        ]
+                    }}
+                }}
+        """
+
+    base_query = [
```

Comment:
I think I found a bug in this query while trying to modify it:

![Scherm­afbeelding 2025-01-15 om 10 25 40](https://github.com/user-attachments/assets/2d20b64d-7ad1-4a6a-b7e1-dd1e4d80ea8d)

Reply:
Right that is the whole point, 1 is really 0 as you are counting the nil